<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationClass</class>
 <widget class="QWidget" name="ApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>-11</y>
     <width>1291</width>
     <height>821</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">background-color: rgba(252,252,252,255)</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>1291</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../logo.jpg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>560</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #CCCCCC;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignInButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>560</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #CCCCCC;</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
